packages/common/src/utils.ts

```diff
@@ -0,0 +1,5 @@
+import type BigNumber from 'bignumber.js'
+
+export function weiToEth(wei: BigNumber): string {
+    return wei.div(1e18).toString(10)
+}
```

Comment:
```suggestion
}

```

Reply:
@taylorjdawson 
- Is there a reason why I should place the extra linespace in line 6?